### PR TITLE
Keep a DRY README.md for wordpress.org & Github

### LINKED ISSUE #406: Add and complete readme.txt
We only need a README.md (not a readme.txt). Let's bring these together into just the README.md and remove the readme.txt.

More info here: https://johnblackbourn.com/readme-md-plugin-on-wordpress-org/

Example of how to format here: https://github.com/johnbillion/user-switching/blob/develop/readme.md

### LINKED ISSUE #610: WP.ORG: Undocumented use of a 3rd Party or external service
We permit plugins to require the use of 3rd party (i.e. external) services, provided they are properly documented in a clear manner.

We require plugins that reach out to other services to disclose this, in clear and plain language, so users are aware of where data is being sent. This allows them to ensure that any legal issues with data transmissions are covered. This is true even if you are the 3rd party service.

In order to do so, you must update your readme to do the following:
Clearly explain that your plugin is relying on a 3rd party as a service and under what circumstances
Provide a link to the service .
Provide a link to the service terms of use and/or privacy policies.
Remember, this is for your own legal protection. Use of services must be upfront and well documented.

Example(s) from your plugin:
# Domain(s) not mentioned in the readme file.
21_21-26-52_gatherpress/includes/core/classes/class-event.php:812 $args = array(
'BEGIN:VCALENDAR',
'VERSION:2.0',
'PRODID:-//GatherPress//RemoteApi//EN',
'BEGIN:VEVENT',
sprintf( 'URL:%s', esc_url_raw( get_permalink( $this->event->ID ) ) ),
sprintf( 'DTSTART:%s', sanitize_text_field( $datetime_start ) ),
sprintf( 'DTEND:%s', sanitize_text_field( $datetime_end ) ),
sprintf( 'DTSTAMP:%s', sanitize_text_field( $datetime_stamp ) ),
sprintf( 'SUMMARY:%s', sanitize_text_field( $this->event->post_title ) ),
sprintf( 'DESCRIPTION:%s', sanitize_text_field( $description ) ),
sprintf( 'LOCATION:%s', sanitize_text_field( $location ) ),
'UID:gatherpress_' . intval( $this->event->ID ),
'END:VEVENT',
'END:VCALENDAR',
);

### PR DESCRIPTION
**Pull request by carstingaxion**
### Description of the Change

Closes #406, Closes #610

### Changelog Entry

> Changed - Keep a DRY README.md for wordpress.org & Github

### Credits

Props @mauteri , @jeffpaul , @patriciabt , @carstingaxion 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/GatherPress/gatherpress/blob/main/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by carstingaxion — COMMENTED** [review]
I'm not 100% sure this will work, but as I understood Mika Epstein correct, using the `wpvideo` shortcode works. But it doesn't look that nice on Github.

Sources:

- https://make.wordpress.org/plugins/2018/03/06/understanding-readme-txt/
- https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#videos
- https://wordpress.org/documentation/article/video-shortcode/

**Comment by carstingaxion** [comment]
I made some screenshots. Which ones, what else and how many should we use?

|  |  |  |  |
|--------|--------|--------|--------|
| Bildschirmfoto vom 2024-05-04 14-52-50 | Bildschirmfoto vom 2024-05-04 14-53-15 | Bildschirmfoto vom 2024-05-04 14-53-28 | Bildschirmfoto vom 2024-05-04 14-53-58 |
| Bildschirmfoto vom 2024-05-04 14-54-11 | Bildschirmfoto vom 2024-05-04 14-55-14 | Bildschirmfoto vom 2024-05-04 14-55-34 | Bildschirmfoto vom 2024-05-04 14-57-03 |
| Bildschirmfoto vom 2024-05-04 14-57-22 | Bildschirmfoto vom 2024-05-04 14-57-31 | Bildschirmfoto vom 2024-05-04 14-57-53 |  |

_And yes_, I'm going to polish the edges before committing the files.

File: README.md [hunk 1]
```diff
@@ -36,27 +46,47 @@ We're creating the very network features we need to host events and gather well.
...
+
+### Credits
+
+Thanks to the following folks for contributing to GatherPress:
+[mauteri](https://profiles.wordpress.org/mauteri/), [hrmervin](https://profiles.wordpress.org/hrmervin/), [pbrocks](https://profiles.wordpress.org/pbrocks/), [jmarx](https://profiles.wordpress.org/jmarx/), [prashantabellad](https://profiles.wordpress.org/prashantabellad/), [prayagm](https://profiles.wordpress.org/prayagm/), [patricia70](https://profiles.wordpress.org/patricia70/), [meaganhanes](https://profiles.wordpress.org/meaganhanes/), [calebthedev](https://profiles.wordpress.org/calebthedev/), [javiercasares](https://profiles.wordpress.org/javiercasares/), [linusx007](https://profiles.wordpress.org/linusx007/), [cameronbarrett](https://profiles.wordpress.org/cameronbarrett/), [courane01](https://profiles.wordpress.org/courane01/), [newyorkerlaura](https://profiles.wordpress.org/newyorkerlaura/), [phoopee3](https://profiles.wordpress.org/phoopee3/), [hauvong](https://profiles.wordpress.org/hauvong/), [deshabhishek007](https://profiles.wordpress.org/deshabhishek007/), [jeffpaul](https://profiles.wordpress.org/jeffpaul/), [michelleames](https://profiles.wordpress.org/michelleames/)
+
```
Comment (mauteri) [inline README.md:64]: Let's make this list just the folks listed as Leads and on Team. See Settings -> Credit. That will keep this list maintainable.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 # GatherPress
 
-> GatherPress, powering our community's event management needs.
+Stable tag: 0.29.0-alpha  
+Tested up to: 6.5.2  
+License: GPL v2 or later  
+Tags: events, event, meetup, community  
+Contributors: mauteri, hrmervin, jmarx, meaganhanes, pbrocks
```
Comment (mauteri) [inline README.md:7]: Let's change this list to same as credits. Just people listed as leads and team under Settings -> Credits (see other comment below).

**Review by mauteri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (carstingaxion) [reply README.md:64]: I would like to see my handle in the list, too.

File: README.md (hunk 2 above)
Comment (carstingaxion) [reply README.md:7]: > I would like to see my handle in the list, too.

Now that I look at `/wp-admin/edit.php?post_type=gatherpress_event&page=gatherpress_credits` you can please ignore my first comment @mauteri .

And a big **thank you** for having me in the team! :man_dancing:  _May I not be your xz contributor._

CC: @patriciabt

Comment (mauteri) [reply README.md]: Looks good!

**Comment by jeffpaul** [comment]
Primary complaint from [Readme Validator](https://wordpress.org/plugins/developers/readme-validator/) are missing screenshots, but that's not necessary for this PR but something certainly worth adding in via a separate PR.  You can take the raw README.md from this PR and paste into [WP Readme](https://wpreadme.com/) to see how things will render on WP.org.

**Comment by carstingaxion** [comment]
The images are here, but not yet decided.

https://github.com/GatherPress/gatherpress/issues/406#issuecomment-2094184821

Thanks for referencing this nice tool and auditing this PR @jeffpaul

**Review by mauteri — APPROVED** [review]
This looks good, @carstingaxion. Thx!

I think I'll also add to the CLI and a unit test to update and compare credits in a future PR so we can manage it in one place and let the script updated it everywhere. If you feel good with this, just move it from draft and we can merge it. Thx again!

**Comment by carstingaxion** [comment]
Thank you @mauteri .

I wanted to wait because @patriciabt and me had some more on the list in #406

And at least some images should be included in the README.

**Comment by carstingaxion** [comment]
But this should not be my decision. I'll open the draft for review and you @mauteri can decide with @patriciabt what to do when. Cheers.

**Comment by mauteri** [comment]
> The images are here, but not yet decided.
> 
> [#406 (comment)](https://github.com/GatherPress/gatherpress/issues/406#issuecomment-2094184821)
> 
> Thanks for referencing this nice tool and auditing this PR @jeffpaul

Cool, @patriciabt can you take a look and let me know if there's other things to add? I'm fine with how it is and we can add more in another PR if need be. Thx!

**Comment by carstingaxion** [comment]
_Maybe we could add another FAQ Question to the bottom of the list, like:_

> What’s about the PRO version?

As a Community powered plugin, GatherPress is already the PRO-version.

Because we strive for close-to-core development, love decisions - not options and follow a lot of well known best-practices within the WordPress space, we can and do focus on what matters most - powering communities.

##### GatherPress‘ best-practices 

_(each element with a relevant link to the standard or tool)_

- tested & validated against WPCS 3
- Consequently checked against the WordPress plugin review guidelines
- ….

**Comment by carstingaxion** [comment]
Oooops. And we missed out totally the _Pilot Proposal._

**Comment by carstingaxion** [comment]
> Oooops. And we missed out totally the _Pilot Proposal._

_(I would suggest adding the following below the first paragraph.)_

>We propose a pilot program to test [GatherPress](https://gatherpress.org/), a community-developed plugin, within interested and active WordPress meetup groups. This initiative stems from our community’s need for an innovative event management tool tailored to the unique demands of WordPress event organizers and participants.
>
>[@Patricia BT](https://profiles.wordpress.org/patricia70/) in January 2024 on
> [make.wordpress.org](https://make.wordpress.org/community/2024/01/22/proposal-pilot-program-to-test-gatherpress-on-the-wordpress-org-network-as-a-meetup-alternative/#comment-31591)

File: README.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ GatherPress, powering our community's event management needs.
 
 GatherPress, a plugin created by and for the WordPress community, is a response to the community's desire for novel event management tools that meet the diverse needs of event organizers and members. Its agenda and roadmap align with that of the WordPress community, ensuring that it evolves in tandem with our collective wants and needs.
 
+**We propose a pilot program to test GatherPress, a community-developed plugin, within interested and active WordPress meetup groups. This initiative stems from our community’s need for an innovative event management tool tailored to the unique demands of WordPress event organizers and participants.**
+[@Patricia BT](https://profiles.wordpress.org/patricia70/) in January 2024 on
+[make.wordpress.org](https://make.wordpress.org/community/2024/01/22/proposal-pilot-program-to-test-gatherpress-on-the-wordpress-org-network-as-a-meetup-alternative/#comment-31591)
```
Comment (mauteri) [inline README.md:27]: Remove `#comment-31591` from URL. Thx!

**Review by mauteri — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -36,27 +49,58 @@ We're creating the very network features we need to host events and gather well.
...
+
+If you wish to share in the collaborative of work to build GatherPress, please drop us a line either via [WordPress Slack](https://make.wordpress.org/chat/) or on [GatherPress.org](https://gatherpress.org/get-involved). The development location of the GatherPress project can be found at [https://github.com/gatherpress/gatherpress](https://github.com/gatherpress/gatherpress). All contributions are welcome: code, design, user interface, documentation, [translation](https://translate.wordpress.org/projects/wp-plugins/gatherpress/) and more.
+
+### Third-Party Libraries
+
+This plugin leverages the following third-party libraries for various functionalities:
+
+- [Moment.js](https://momentjs.com/): Used for date and time manipulation.
+- [Moment-Timezone](https://momentjs.com/timezone/): Adds timezone support to Moment.js.
+- [Playwright](https://playwright.dev/): Enables browser automation for testing and interaction.
```
Comment (mauteri) [inline README.md:69]: Not needed here as it's not packaged with the plugin.

**Comment by mauteri** [comment]
@carstingaxion @patriciabt @jeffpaul is there anything else that needs to be done in the README.md or any questions you have or can this be merged? Thx!

**Comment by mauteri** [comment]
> @carstingaxion @patriciabt @jeffpaul is there anything else that needs to be done in the README.md or any questions you have or can this be merged? Thx!

I added one last comment on this.

**Comment by mauteri** [comment]
Made some changes and going to merge this. Let's review the README.md now completely rendered in GH and if there's any additional changes let's put in follow up PRs. Thx!

**Review by mauteri — APPROVED** [review]

**Pull request merged**